platform/darwin/src/MGLStyle.mm

```diff
@@ -448,7 +449,7 @@ - (void)insertLayer:(MGLStyleLayer *)layer aboveLayer:(MGLStyleLayer *)sibling {
     if (index + 1 > layers.size()) {
         [NSException raise:NSInvalidArgumentException
                     format:
-         @"A style layer cannot be placed above %@ in the style. "
+         @"A style layer cannot be placed below %@ in the style. "
```

Comment:
Oops, I misread the diff when I suggested rewording this to say “below” instead of “above” in https://github.com/mapbox/mapbox-gl-native/pull/7484#issuecomment-268092473. It should indeed say “above”, because that’s what this method is all about.